source/aks/patching-neuvector.html.md.erb

```diff
@@ -0,0 +1,218 @@
+
+Using Git you need to:
+
+- Clone the [HMCTS Neuvector chart](https://github.com/hmcts/chart-neuvector) repo
+- Create your own branch to carry out the work
+- Update the [HMCTS Neuvector dependency version](https://github.com/hmcts/chart-neuvector/blob/master/neuvector-azure-keyvault/Chart.yaml#L9-L12) to have the latest upstream release version e.g. v2.7.3 -> v2.7.4
+- Commit and push your changes and raise a PR for review
+
+Once reviewed and merged you need to draft a new release with the latest code.
+You can find steps [here on how to draft a release](../../Testing-Changes/drafting-a-release.html#create-a-pre-release-with-your-branch-as-the-source-and-a-new-tag)
+
+The instructions on that page are specific to draft releases, in this case you can create a new `latest release` by using the next version number available and setting `set as the latest release` before selecting `Publish release`.
+
+This will trigger the [Azure DevOps pipeline](https://dev.azure.com/hmcts/PlatformOperations/_build?definitionId=799) to run and validate and release a new version.
+
+## Non-Production Patching
+
+### Check for additional upstream changes
+
+**_This section is only necessary for non-production environment patching_**
+
+Part of the Neuvector setup involves the installation of CRDs (custom resource definitions) and these may need to be updated as well.
+The versions installed depend on the version of the App within the upstream Neuvector chart that is now being installed
+
+- [Neuvector CRD manifests](https://github.com/neuvector/manifests/tree/main/kubernetes)
+- [Version of the app being installed](https://github.com/neuvector/neuvector-helm/blob/master/charts/crd/Chart.yaml#L4)
+
+The currently installed CRDs can be found in the [Flux apps folder for Neuvector](https://github.com/hmcts/sds-flux-config/blob/master/apps/neuvector/crds/kustomization.yaml)
+
+Pay close attention to the version shown in the URLs `/5.2.0/` which denotes the app version.
+
+If the version is changing, you can install the latest versions per AKS cluster by creating a new folder in the Neuvector directory with a new `kustomize.yaml` file and add the latest manifest links to this file:
+
+![neuvectorCRDs](Images/NeuvectorUpdatedCRDs.png)
+
+You will also need to copy the existing [`base/kustomize.yaml`](https://github.com/hmcts/sds-flux-config/blob/master/apps/neuvector/base/kustomize.yaml) file into this new directory so you can point to your newly created file.
+
+Update the path to point to your new directory `path: ./apps/neuvector/crds` -> `path: ./apps/neuvector/<some name>`
+
+With these files now in place you can patch any specific AKS cluster by adding a patch to the clusters [base kustomization](https://github.com/hmcts/sds-flux-config/blob/master/clusters/ithc/base/kustomization.yaml) file:
+
+```yaml
+  - path: ../../../apps/neuvector/patching-crds/kustomize.yaml`
+```
```

Comment:
Could you link to the PRs you made here as reference? Might be a bit confusing to follow solely based on this